app/assets/main/react_components/context/Footprint/TextsContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+
+const TextsContext = React.createContext(undefined);
+export const useTexts = () => {
```

Comment:
Tillhandahåller all textdata.

Säg exempelvis att ytterligare en textfil vill användas. Då adderar vi bara ett propsvärde och lägger in i variabeln const texts (nedan), så har alla komponenter tillgång till den texten.